bootstrap_cfn/fab_tasks.py

```diff
@@ -18,7 +15,10 @@
 from bootstrap_cfn.cloudformation import Cloudformation
 from bootstrap_cfn.config import ConfigParser, ProjectConfig
 from bootstrap_cfn.elb import ELB
-from bootstrap_cfn.errors import BootstrapCfnError, CfnConfigError, CloudResourceNotFoundError, DNSRecordNotFoundError, ZoneIDNotFoundError
+from bootstrap_cfn.errors import ActiveTagExistConflictError, BootstrapCfnError,\
```

Comment:
preferred way to write long import line is to use `()` rather than `\` https://www.python.org/dev/peps/pep-0328/

```
from bootstrap_cfn.errors import (ActiveTagExistConflictError, BootstrapCfnError,
    CfnConfigError, CloudResourceNotFoundError, DNSRecordNotFoundError,
    ...)
```